docs/fastdds/use_cases/wide_deployments/wide_deployments.rst

```diff
@@ -0,0 +1,38 @@
+.. _wide_deployments:
+
+Wide Deployments
```

Comment:
Although I agree that it also applies to wifi, I feel it shouldn't be as a subchapter for wifi, as it can be used in many other situations. I kept it as a separate use case, but referenced from the wifi use case.